corpus_raw.conllulex

```diff
@@ -19298,7 +19298,7 @@
 15	,	,	PUNCT	,	_	19	punct	_	_	_	_	_	_	_	_	_	_	_
 16	one	one	NUM	CD	_	19	nummod	_	_	_	_	_	_	_	_	_	_	_
 17	-	-	PUNCT	HYPH	_	16	punct	_	_	_	_	_	_	_	_	_	_	_
-18	off	off	ADP	RP	_	16	compound	_	_	_	_	_	_	_	_	_	_	_
+18	off	off	ADV	RB	_	16	compound	_	_	_	_	_	_	_	_	_	_	_
```

Comment:
"one-off" is a tricky expression. `compound` is usually right-headed. Would `advmod` work?

I think `nummod` is incorrect for "one". Probably it should attach as `compound`.